### DISCUSSION TITLE
ImportError: cannot import name 'LoadPNG' from 'monai.transforms'

### DISCUSSION THREAD

**Discussion by Mitra-coding** [Project-MONAI/MONAI#2528] [Q&A]
Hi,
 I've installed monai in colab:
`!pip install monai`

when I want to import  'LoadPNG':
`from monai.transforms import LoadPNG
`

I get this error:
`ImportError: cannot import name 'LoadPNG' from 'monai.transforms' (/usr/local/lib/python3.7/dist packages/monai/transforms/__init__.py)`

Upgrading via
`!pip install --upgrade monai
`didn't help.

Any guidance would be greatly appreciated.

**Comment by wyli** [comment]
`LoadPNG` is deprecated, please use `!pip install monai[pillow]`, and `LoadImage`:
```python
from monai.transforms import LoadImage
LoadImage(reader="pilreader")(filename='test.png')
```
